tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -32,11 +34,18 @@ const getReportTaskFiltersText = async () => {
   return blah;
 };
 
+const getAllContactRows = async () => {
```

Comment:
I know this is a mistake I added but can you not repeat it here.  Since the code is so similar. I would re-factor this so there is a generic function that takes the elements and returns the text. Then have two functions that just pass in the expected elements. 

```
const getTextForElements(elements) => {
  const texts = await Promise.all((await elements()).map(filter => filter.getText()));
  return texts ;
}

const getReportTaskFiltersText = async () => {
  await (await taskFilter()).waitForDisplayed();
  return getTextForElements(taskFilters());
};
```

Reply:
Done